tests/integration/source_code/test_jobs.py

```diff
@@ -34,10 +34,10 @@
 
 
 @retried(on=[NotFound], timeout=timedelta(minutes=5))
-def test_running_real_workflow_linter_job(installation_ctx, make_notebook, make_directory, make_job):
+def test_running_real_workflow_linter_job(installation_ctx, make_notebook, make_directory, make_job) -> None:
     # Deprecated file system path in call to: /mnt/things/e/f/g
     lint_problem = b"display(spark.read.csv('/mnt/things/e/f/g'))"
-    notebook = make_notebook(path=f"{make_directory()}/notebook.ipynb", content=lint_problem)
+    notebook = make_notebook(path=f"{make_directory()}/notebook.py", content=lint_problem)
     job = make_job(notebook_path=notebook)
```

Comment:
With the changes from [this PR](https://github.com/databrickslabs/pytester/pull/60), this would become:

`job = make_job(content="display(spark.read.csv('/mnt/things/e/f/g'))")`